### ISSUE TITLE
Command Line Interface script

### ISSUE THREAD

**Issue by antononcube** [kubaPod/M2MD#25]
Please modify the package M2MD, so Command Line Interface (CLI) scripts can be made.

For example the following script does not work because M2MD requires front-end operations:

```
#!/usr/bin/env wolframscript -function

Block[{},
  Needs["M2MD`"];

  If[FileExistsQ[#1],
   fileName = StringReplace[FileNameSplit[#1][[-1]], {".nb" ~~ EndOfString -> ".md"}];

   SeedRandom[2323];

   M2MD`MDExport[fileName, NotebookRead[#1]];
   SystemOpen[fileName],

   (*ELSE*)
   Print["File does not exist."]]
 ] &
```

**Comment by saltynexus** [comment]
I'm going to second this request. There seems to be growing interest in Jupyter Notebooks + Wolframe Engine, but the ability to convert Mathematica notebooks to Jupyter notebooks does not exist without having access to Mathematica. Anton has provided a solution here: 

https://community.wolfram.com/groups/-/m/t/2562963

but points out that this tool (M2MD) requires Mathematica. What he is requesting here might be the work around and is worth further investigation.

**Comment by DrRIH** [comment]
Agree, the case of most interest is for people working with Wolfram Engine and Jupyter. Requiring a full Mathematica Front-end defeats the purpose. Access to a Wolfram Kernel via Wolfram Engine or Wolfram Cloud seems acceptable, but probably one should avoid duplicating efforts and take advantage of development like "Pandoc" https://pandoc.org .

Probably, if the developer could spell out what is the main issue that the Front-end is solving we could help find a way around this dependency. The programming challenge could be posted in Mathematica Stackexchange, a much superior place for curated expert knowledge than community.wolfram.com, which seems to be quite chatty and light in content.
